test/views/TrusteesNameViewSpec.scala

```diff
@@ -19,27 +19,36 @@ package views
 import controllers.routes
 import forms.TrusteesNameFormProvider
 import generators.Generators
-import models.{FullName, NormalMode}
+import models.{FullName, NormalMode, UserAnswers}
+import pages.{IsThisLeadTrusteePage, TrusteesNamePage}
 import play.api.data.Form
+import play.api.i18n.Messages
 import play.twirl.api.HtmlFormat
 import views.behaviours.QuestionViewBehaviours
 import views.html.TrusteesNameView
+import views.html.components.heading
 
 class TrusteesNameViewSpec extends QuestionViewBehaviours[FullName] with Generators {
 
   val messageKeyPrefix = "trusteesName"
 
+  val heading = Messages("trusteesName.heading")
+
   val index = 0
 
   override val form = new TrusteesNameFormProvider()()
 
-  "TrusteeFullName view" must {
+  "LeadTrusteeFullName view" must {
```

Comment:
You've got LeadTrusteeFullName but your heading is always `trusteesName.heading` for a standard trustee?